packages/jumpstarter-driver-can/README.md

```
@@ -18,17 +25,120 @@ export:
+      params:
+        max_frame_size: 2048
+        blocking_send: false
+        can_fd: true
+
+```
+
+| Parameter     | Description | Type | Required | Default |
+| --------------| ----------- | ---- | -------- | ------- |
+| interface     | Refer to the [python-can](https://python-can.readthedocs.io/en/stable/interfaces.html) list of interfaces | `str`  | yes      |         |
+| channel       | channel to be used, refer to the interface documentation | `int` or `str` |        |     |
+| address       | Refer to the [isotp.Address](https://can-isotp.readthedocs.io/en/latest/isotp/addressing.html#isotp.Address) documentation | isotp.Address | yes |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)